App.js

```diff
@@ -31,6 +31,12 @@ class App extends React.Component {
     }
   }
 
+  makeListItem(textToAdd) {
```

Comment:
Yeah it was part of an attempt to incorporate the list tooltip functionality, and I forgot to get rid of it, hence all of these problems